tools/preload.py

```diff
@@ -63,6 +63,8 @@ def get_absolute_url(domain, path, icon):
 
     if(path.startswith('http')):
         return icon_path
+    elif path_absolute_from_app_root:
```

Comment:
I feel like the name says the reverse of what it does. Shouldn't it be called `path_relative_from_app_root`? Then, as it is relative, we make it absolute by appending the missing part, on line 67. WDYT?